rust/lance/src/dataset/scanner.rs

```diff
@@ -1626,9 +1598,13 @@ impl Scanner {
 
             // Find all deltas with the same index name.
             let deltas = self.dataset.load_indices_by_name(&index.name).await?;
-            let ann_node = self.ann(q, &deltas, filter_plan).await?; // _distance, _rowid
+            let (ann_node, is_multivec) = match vector_type {
+                DataType::FixedSizeList(_, _) => (self.ann(q, &deltas, filter_plan).await?, false),
+                DataType::List(_) => (self.multivec_ann(q, &deltas, filter_plan).await?, true),
+                _ => unreachable!(),
+            };
 
-            let mut knn_node = if q.refine_factor.is_some() {
+            let mut knn_node = if q.refine_factor.is_some() || is_multivec {
```

Comment:
for multivector, refine is always required

Reply:
i dont follow, why is it?

Reply:
this just follows the algo that colbert paper described, this is required for calculating the maxsim distance. without refine, the search just finds nearest chunks without considering maxsim metric